components/avatar.css

```diff
@@ -0,0 +1,50 @@
+/* common class for all avatars for border-radius property */
+
+.avatar {
+    border-radius: 50%;
+}
+
+
+/* class for avatars container div */
+
+.img-avatar {
+    padding: 1rem;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    align-items: flex-end;
+    gap: 2rem;
+    margin-bottom: 1rem;
+}
```

Comment:
Avatar will break for images that are not square I think, could add a fix for that.